images/singleuser/install-pwb

```diff
@@ -4,6 +4,9 @@ set -e
 # Remove 'Pillow' from requirements.txt
 sed -e '/Pillow/d' /srv/paws/pwb/requirements.txt > /tmp/requirements.txt
 
+# To prevent breakage on older python
+/srv/paws/bin/pip install --no-cache-dir -U pip setuptools
```

Comment:
It might be! The build just kind of mentions that wheel isn't installed. I don't know if it would speed it up much, but it could. We could also pass it off to the next iteration.

Reply:
So this got me on a sidebar, and I tried upgrading the container. With Ubuntu Focal, it'd have python 3.8 (which would make our users happy), and it would drop python 2 entirely (...not sure there, but there is no python 2 kernel in PAWS anyway), AND it requires upgrading node to a modern version (because the ancient version in here requires python2 to install lol) which is much more destructive than a python upgrade. I think upgrading the container definitely is another PR.

So! Installing wheel is needed here or you cannot build wheels (tested in a container), and while users can totally install wheel on their own we can install it.  When I started on that, I noticed that there are duplicates of the deprecated libraries I replaced on this PR in another script (fixed in a commit coming shortly), which may explain some odd behavior I saw, also there's a direct git install of nbresuse instead of using pypi, so I switched to the pypi version. There is now only one personal repo of Yuvi's in here, and that one needs to stay or be handed off later (python-wdqs).

Reply:
On fixing the fact that some of the old stuff from Yuvi's repos were still lurking, something broke. I cannot reach public notebooks to do `import paws.YuviPanda.TeaHouse` like I could because it gets a handshake failure. The handshake failure is almost certainly this problem: https://phabricator.wikimedia.org/T257534#6343260  That is likely to bite me in more places if I don't figure it out.

Reply:
Done in 79a6b9e3e9c8, but now we have the problem that VMs don't seem to trust LE certs.

Reply:
And now they do! This works just as well as it actually can again: https://public.paws.wmcloud.org/User:BStorm_(WMF)/Untitled.ipynb